docs/reference/portal.dataProduct.json

```diff
@@ -595,6 +595,54 @@
                                     }
                                 }
                             ]
+                        },
+                        {
+                            "name": "monitoringSettings",
+                            "label": "Monitoring Settings",
+                            "type": "Microsoft.Common.Section",
+                            "visible": true,
+                            "elements": [
+                                {
+                                    "name": "monitoringSettingsText",
+                                    "type": "Microsoft.Common.TextBlock",
+                                    "visible": true,
+                                    "options": {
+                                        "text": "Specify Monitoring settings for this deployment.",
+                                        "link": {
+                                            "label": "",
+                                            "uri": ""
+                                        }
+                                    }
+                                },
+                                {
+                                    "name": "enableMonitoring",
+                                    "label": "Deploy Monitoring Components",
+                                    "type": "Microsoft.Common.OptionsGroup",
+                                    "visible": true,
+                                    "toolTip": "Deploy monitoring components like Log Analytics, Azure dashboards, alerts and diagnostic settings.",
+                                    "defaultValue": "No",
+                                    "constraints": {
+                                        "allowedValues": [
+                                            {
+                                                "label": "Yes",
+                                                "value": "yes"
+                                            },
```

Comment:
```suggestion
                                    "defaultValue": "Yes (recommended)",
                                    "constraints": {
                                        "allowedValues": [
                                            {
                                                "label": "Yes (recommended)",
                                                "value": "yes"
                                            },
```

Reply:
I guess we should enable this by default?

Reply:
Fixed